cmake/Buildbipedal-locomotion-framework.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2020  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Giulio Romualdi <gulio.romualdi@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+
+set(bipedal-locomotion-framework_DEPENDS "")
+list(APPEND bipedal-locomotion-framework_DEPENDS YARP)
+list(APPEND bipedal-locomotion-framework_DEPENDS iDynTree)
+
+if (ROBOTOLOGY_ENABLE_DYNAMICS_FULL_DEPS)
+  include(FindOrBuildPackage)
+
+  find_or_build_package(manif QUIET)
+  find_or_build_package(qhull QUIET)
+  find_or_build_package(CppAD QUIET)
+  find_or_build_package(casadi QUIET)
+
+  list(APPEND bipedal-locomotion-framework_DEPENDS manif)
+  list(APPEND bipedal-locomotion-framework_DEPENDS qhull)
+  list(APPEND bipedal-locomotion-framework_DEPENDS CppAD)
+  list(APPEND bipedal-locomotion-framework_DEPENDS casadi)
+
+endif()
+
+ycm_ep_helper(BipedalLocomotionFramework TYPE GIT
+              STYLE GITHUB
+              REPOSITORY dic-iit/bipedal-locomotion-framework.git
+              TAG master
+              COMPONENT dynamics
+              FOLDER robotology
+              CMAKE_ARGS -DBUILD_TESTING:BOOL=OFF
```

Comment:
Could it make sense to pass the options on enabling/disabling the dependencies explicitly?

Reply:
do you mean passing `FRAMEWORK_USE_manif` etc? https://github.com/dic-iit/bipedal-locomotion-framework/blob/master/cmake/BipedalLocomotionFrameworkFindDependencies.cmake#L179

Reply:
https://github.com/robotology/robotology-superbuild/pull/526/commits/ab9935780819c036379b14c3e5d08cab9245bbd3